corehq/apps/accounting/static/accounting/js/payment_method_handler.js

```diff
@@ -416,6 +416,7 @@ hqDefine('accounting/js/payment_method_handler', [
         self.features = data.features;
         self.general_credit = data.general_credit;
         self.invoice_credit = data.invoice_credit;
+        self.invoice_credit_quantity = data.invoice_credit().quantity;
```

Comment:
is this quantity an observable? should this be a `ko.computed`?

Reply:
Yes it should (though it worked as written because I was calling `self.invoice_credit_quantity()`). Realized I probably don't even need this assignment since I'm only using it in one place where `self.invoice_credit` is already accessible.

Reply:
https://github.com/dimagi/commcare-hq/pull/36064/commits/4a6076ed8793992dfa1ffca5833eba97eeb02ebc